backend/game_server/lib/helpers/physic.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Helpers.Physic do
+	import Helpers.Vector
+	alias  Helpers.Vector, as: Vector
+
+	def move(%Vector{} = position, %Vector{} = velocity, time) do
+		position |> sum( velocity |> scale( time ) )
```

Comment:
http://elixir-lang.org/getting-started/alias-require-and-import.html

If you are using import you can write functions without methods origins. 
But structs still need be fullnamed so i create alias  Helpers.Vector, as: Vector  to use struct  %Vector{} instead of %Helpers.Vector{}